### PR TITLE
 #185172080 Add a readme file and badges

### PR DESCRIPTION
**Pull request by AdrineUWERA**
#### What does this PR do?

Adds a readme file with badges. 

#### Description of Task to be completed?

Add a detailed readme file for the API to include the information below:

- Installation and run commands
- Description
- Dependencies
- Tests setup and execution
- Add badges:
    Code Climate maintainability badge
    Code Climate code coverage badge
    CircleCI status badge

#### How should this be manually tested?

- Check the repo's readme

#### What are the relevant pivotal tracker/Trello stories?

#185172080

### CODE REVIEW

**Review by Kress2000 — APPROVED** [review]

**Review by Edcreation — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  run_setup:
+    docker:
+      - image: circleci/node:16
+
+    steps:
+      - checkout
+      - run:
+          name: Install npm dependencies
+          command: npm install
+
```
Comment (Edcreation) [inline .circleci/config.yml:12]: We are using yarn

**Review by Edcreation — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,45 +1,153 @@
...
+- To lint the source code with ESLint run `npm run lint`.
+
+### Testing
+
+The e-commerce frontend includes a comprehensive test suite to ensure the reliability and correctness of the code. The tests are written using the Jest testing framework and React Testing Library for testing React components.
+
+To run the tests, execute the following command:
+
+```bash
+npm run test
```
Comment (Kress2000) [inline README.md:65]: were are using yarn not npm

**Review by Kress2000 — CHANGES_REQUESTED** [review]
unless you changed the package.js and package-loc, we are using yarn plz! check there

**Review by Edcreation — APPROVED** [review]

**Review by Kress2000 — APPROVED** [review]
good, but I was suggesting to  add `yarn run dev --host` in the ### Getting started  section  for those who want to start the project on IP address. For this method you can even access it on your phone, it it will be good for us in development during responsiveness

**Review by Pacific911 — APPROVED** [review]

**Review by munezerobagira — APPROVED** [review]

**Pull request merged**